### PR TITLE
BACK-842 : Ressource utils for generic retry logic (linear / Backoff)

### PR DESCRIPTION
**Pull request by RomanWlm**

### CODE REVIEW

**Review by nukky94 — APPROVED** [review]
Nice 👍

**Comment by RomanWlm** [comment]
Hodl !!!
Brand new CI is complaining :D

File: common/src/main/scala/co/ledger/lama/common/utils/RabbitUtils.scala [hunk 1]
```diff
@@ -25,21 +26,22 @@ object RabbitUtils {
...
-    }
+  )(implicit t: Timer[IO]): IO[Unit] =
+    retriableResource(R.createConnectionChannel, RetryPolicy.exponentialBackOff())
+      .use { implicit channel =>
+        exchanges
+          .map {
+            case (exchangeName, exchangeType) =>
+              R.declareExchange(exchangeName, exchangeType)
+          }
+          .sequence
```
Comment (vbergeron-ledger) [inline common/src/main/scala/co/ledger/lama/common/utils/RabbitUtils.scala:42]: map + sequence = traverse

**Review by vbergeron-ledger — APPROVED** [review]

**Comment by estrauser-ledger** [comment]
Ci failed because no llama was provided

**Pull request merged**